saleor/graphql/attribute/utils.py

```diff
@@ -640,28 +724,38 @@ def validate_attributes_input(
 
     error_code_enum = PageErrorCode if is_page_attributes else ProductErrorCode
     attribute_errors: T_ERROR_DICT = defaultdict(list)
+    input_type_to_validation_func_mapping = {
+        AttributeInputType.BOOLEAN: validate_boolean_input,
+        AttributeInputType.DATE: validate_date_time_input,
+        AttributeInputType.DATE_TIME: validate_date_time_input,
+        AttributeInputType.DROPDOWN: validate_dropdown_input,
+        AttributeInputType.FILE: validate_file_attributes_input,
+        AttributeInputType.NUMERIC: validate_numeric_input,
+        AttributeInputType.MULTISELECT: validate_multiselect_input,
+        AttributeInputType.PLAIN_TEXT: validate_plain_text_attributes_input,
+        AttributeInputType.REFERENCE: validate_reference_attributes_input,
+        AttributeInputType.RICH_TEXT: validate_rich_text_attributes_input,
+    }
```

Comment:
Same with validation, it seems like it should be extracted maybe to a separate file.